src/main/java/org/galatea/starter/entrypoint/PriceRestController.java

```diff
@@ -35,19 +36,20 @@
    *
    * @GetMapping to link http GET request to this method
    * @RequestParam to take a parameter from the url
+   * @return
    */
 
 
   @GetMapping(value = "${webservice.halpath}", produces = {MediaType.APPLICATION_JSON_VALUE})
-  public AlphaPrices priceEndpoint(
+  public Collection<StockPrices> priceEndpoint(
 
   @RequestParam(value = "stock") String stock,
       @RequestParam(value = "days", defaultValue= "1", required = false) String daysToLookBack,
       @RequestParam (value = "requestId", required = false) String requestId)
 
       throws IOException, SQLException {
     processRequestId (requestId);
-
-    return priceService.getPricesByStock(stock, daysToLookBack);
+    priceService.getPricesByStock(stock, daysToLookBack);
+    return priceService.ConvertPrices(stock);
```

Comment:
put the convertPrices() method call inside priceService.getPricesByStock(). have it be the next step after you have fetched and filtered the data from AlphaVantage